src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerListCriteria.kt

```diff
@@ -1,13 +1,17 @@
 package uk.gov.justice.digital.hmpps.prisonersearch.services
 
 import io.swagger.v3.oas.annotations.media.Schema
+import javax.validation.constraints.NotEmpty
+import javax.validation.constraints.Size
 
 @Schema(description = "Search Criteria for a list of prisoners")
 
 data class PrisonerListCriteria(
   @Schema(description = "List of prisoner numbers to search by", example = "[\"A1234AA\"]")
+  @NotEmpty
+  @Size(min = 1, max = 1000)
```

Comment:
couldn't get this validation to actually be applied, but worked well for the swagger docs :grinning: